CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+# to-be-released
+
+## Changed
+
+- [BREAKING] Internal representation of hash schemas was changed to be a simple list of key types (flash-gordon)
+  `Dry::Types::Hash#with_type_transform` now yields a key type instead of type + name:
+  ```ruby
+  Dry::Types['stirct.hash'].with_type_transform { |key| key.name == :age ? key.required(false) : key }
+  ```
+
+## Added
+
+- Optional keys for schemas can be provided with ?-ending symbols (flash-gordon)
```

Comment:
Do we want to put an explainer in here for how to use ?-ending names when we _don't_ want the implicit optional key behaviour?

Reply:
actually, I had no plans for that. I'd like to see a real use case for that first